corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -1302,19 +1293,12 @@ def get_properly_wrapped_sync_log(doc_id):
     """
     try:
         synclog = SyncLogSQL.objects.filter(synclog_id=doc_id).first()
-        if synclog:
-            return properly_wrap_sync_log(synclog.doc)
+        return properly_wrap_sync_log(synclog.doc)
```

Comment:
I think this could raise an error if `synclog` is None. What about doing `SyncLogSQL.objects.get(synclog_id=doc_id)` and catching the `DoesNotExist`?